web/src/main/java/se/inera/intyg/webcert/web/web/controller/integration/IntygIntegrationController.java

```diff
@@ -498,5 +515,25 @@ private void validateRequest(Map<String, Object> pathParameters) {
         super.validateParameters(pathParameters);
         super.validateAuthorities();
     }
+
+    private boolean launchIdShouldBeAdded(String launchId) {
+        if (launchId == null) {
+            return false;
+        } else {
+            return !launchId.isEmpty();
+        }
+    }
+
+    private String assertLaunchIdFormat(@NotNull String launchId) {
```

Comment:
This method still does two things. It asserts and returns the launchId to add. Why isn't the validation of the launchId done in "launchIdShouldBeAdded"?